MODULE.bazel

```diff
@@ -23,12 +23,32 @@ use_repo(
 
 register_toolchains("//appimage:all")
 
+PYTHON_VERSIONS = [
+    "3.8",
+    "3.9",
+    "3.10",
+    "3.11",
+]
```

Comment:
Wouldn't this have to match all the versions in
https://github.com/bazelbuild/rules_python/blob/4c2d7d9d6608795522322a9b2294b5053e41b979/python/versions.bzl#L47-L48

Reply:
As far as I understand, this is not really about toolchains. It is about pip wheels. The pip wheels need to be registered for all python versions to be supported. I am not sure if those wheels really care about the patch version of python.